doc/source/architecture.rst

```diff
@@ -131,9 +137,10 @@ It consists of the following tools:
 
 * **Orchestrator** is a MySQL replication topology management
```

Comment:
Can we mention that Orchestrator is a standalone tool, not integrated with PMM Client, we added it as experimental? Also mention in the release notes 1.0.5.